components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/workflow/TenantWorkflowConfigHolder.java

```diff
@@ -54,6 +54,9 @@
 
     private static final QName ATT_NAME = new QName("name");
 
+    private static final String SUBSCRIPTION_UPDATE_EXECUTER_CLASS =
+            "org.wso2.carbon.apimgt.impl.workflow.SubscriptionUpdateSimpleWorkflowExecutor";
+
     private transient SecretResolver secretResolver;
```

Comment:
Let's change to DEFAULT_SUBSCRIPTION_UPDATE_EXECUTER_CLASS

Reply:
resolved